dist/openshift/cincinnati.yaml

```diff
@@ -174,7 +176,7 @@ objects:
           volumes:
             - name: secrets
               secret:
-                secretName: cincinnati-registry-credentials
+                secretName: cincinnati-credentials
```

Comment:
`hack/e2e.sh` is still using `cincinnati-registry-credentials`.

I think this needs two template options:
* CREDENTIALS_SECRET_NAME (set by default to stage github token secret)
* CINCINNATI_GITHUB_SCRAPER_OAUTH_TOKEN_PATH (set to github token path by default)

and have `hack/e2e.sh` adjusted to create a dummy secret and pass `-p CINCINNATI_GITHUB_SCRAPER_OAUTH_TOKEN_PATH=""` in `oc new-app`.

This way we can have:
* CREDENTIALS_SECRET_NAME in PR e2e set to docker credentials and CINCINNATI_GITHUB_SCRAPER_OAUTH_TOKEN_PATH set to none. e2e test for PRs won't use github token for authentication
* stage / prod deploy would mount the secret and use token-based authentication

This is similar to RUST_BACKTRACE env var

Reply:
Since `CINCINNATI_GITHUB_SCRAPER_OAUTH_TOKEN_PATH=""` will lead to the error, I tried to find a way to leave this variable unset in the e2e environment. Please see the latest commit and let me know if it's a viable approach or not.

Reply:
That would work, but wouldn't template look cleaner if Cincinnati support empty path in `CINCINNATI_GITHUB_SCRAPER_OAUTH_TOKEN_PATH`?

Reply:
In my mind that's a workaround and I would prefer to indicate the absence of the secret by the absence of the environment variable, instead of special casing the `""` value.

If you feel strongly against this exercise I can be convinced otherwise.